build_runner/lib/src/server/graph_viz_main.dart

```diff
@@ -15,21 +16,42 @@ main() async {
   searchForm.onSubmit.listen((e) async {
     e.preventDefault();
     var query = searchBox.value;
-    _focus(query);
+    await _focus(query);
```

Comment:
could also make the method not async instead

Reply:
true!